content/docs/command-reference/dag.md

```diff
@@ -0,0 +1,108 @@
+# dag
+
+Show [stages](/doc/command-reference/run) in a pipeline that lead to the
+specified stage. By default it lists
+[DVC-files](/doc/user-guide/dvc-files-and-directories).
+
+## Synopsis
+
+```usage
+usage: dvc dag [-h] [-q | -v] [--dot] [--full] [target]
+
+positional arguments:
+  targets         Stage or output to show pipeline for (optional)
+                  Finds all stages in the workspace by default.
+```
+
+## Description
+
+A data pipeline, in general, is a series of data processing
+[stages](/doc/command-reference/run) (for example console commands that take an
+input and produce an <abbr>output</abbr>). A pipeline may produce intermediate
+data, and has a final result. Machine learning (ML) pipelines typically start a
+with large raw datasets, include intermediate featurization and training stages,
+and produce a final model, as well as accuracy
+[metrics](/doc/command-reference/metrics).
+
+In DVC, pipeline stages and commands, their data I/O, interdependencies, and
+results (intermediate or final) are specified with `dvc add` and `dvc run`,
+among other commands. This allows DVC to restore one or more pipelines of stages
+interconnected by their dependencies and outputs later. (See `dvc repro`.)
+
+> DVC builds a dependency graph
+> ([DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) to do this.
+
+`dvc dag` displays the stages of a pipeline up to the target stage. If `target`
+is omitted, it will show the full project DAG.
```

Comment:
I think we shuold start the description with this paragraph though. Since it's now a specific command after all 🙂

Reply:
Never mind, the short intro is enough. That one needs updating though... Will do.